# exercise 3.2.4 from unit 3
'''
Here is a code snippet and the output obtained after running it.

The code section uses a function named divider, which accepts as parameters two numbers (a and b) and prints the result of division between them.

def divider(a, b):
    try:
        result = a / b
        print(result)
    except ________________:
        print("print me!")
    except ZeroDivisionError:
        print("don't print me!")
divider(5, 0)
print me!

Complete the missing exception type in the program (highlighted with a marker) so that the requested output is obtained.
'''
# Answer: Exception 
